File: Praktikum 2. Input, dan output/Soal 3/PRAK203-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>

int main()
{
    double a, b, i, j, x, y;
    double hasil;

    printf("Masukkan nilai a, b, i, j, x, dan y (secara berurutan): ");
    scanf("%lf %lf %lf %lf %lf %lf", &a, &b, &i, &j, &x, &y);

    hasil = ((a - b) * (i / j)) - (x + y);

    printf("%.3lf\n", hasil);

    return 0;
}

File: Praktikum 2. Input, dan output/Soal 3/PRAK203-2310817310001-AvantioFierzaPatria.py
a, b, i, j, x, y = map(float, input("Masukkan nilai a, b, i, j, x, dan y (secara berurutan): ").split())

hasil = ((a - b) * (i / j)) - (x + y)

print("{:.3f}".format(hasil))
